docs/recipes/git-show.rst

```diff
@@ -40,6 +40,22 @@ Show all files in commit
     >>> for e in commit.tree:
     >>>     print(e.name)
 
+======================================================================
+Produce something like a `git show` message
+======================================================================
+
+    >>> from __future__ import unicode_literals
+    >>> from datetime import datetime
+    >>> import pytz
+    >>> tzinfo  = pytz.timezone('Europe/Berlin')
```

Comment:
The timezone (really offset) is not fixed for the user running git-show, but is read from the commit, in this case from `commit.author.time_offset`.